SwiftLoggerSampleApp/SwiftLoggerSampleApp/Actions.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+
+class ActionsViewModel: ObservableObject {
```

Comment:
Mas tu takovou zvlastni nekonzistenci. Chapu, ze mas pro zjednoduseni VM i View ve stejnym souboru. Ale mas soubor `AppView` a pak soubor `Actions`. To bych teda vubec necekal, ze jsou oba dva soubory feature. Asi bych to zmenil na `AppFeature` a `ActionsFeature`.

Reply:
A mozna uz bych to rozlozil do slozek, kdyz ses rozhodl tu Sample appku udelat takhle vetsi...

Reply:
Good point, upraveno ✔